go.mod

```diff
@@ -3,22 +3,32 @@ module github.com/brave/nitriding
 go 1.19
 
 require (
-	github.com/brave/viproxy v0.1.2
+	github.com/containers/gvisor-tap-vsock v0.4.0
+	github.com/go-chi/chi v1.5.4
 	github.com/go-chi/chi/v5 v5.0.7
```

Comment:
Do we need both `chi` and `chi/v5` here? The [landing page](https://github.com/go-chi/chi) suggests
```go
import (
        "github.com/go-chi/chi/v5"
        "github.com/go-chi/chi/v5/middleware"
)
```

Reply:
Good catch. Fixed in a832419.